configs/train/pretrain/mr_ct_config.yaml

```diff
@@ -0,0 +1,77 @@
+main:
+    random_seed: 0
+    saved_dir: './models/pretrain/mr_ct'
+
+dataset:
+    name: 'PretrainDataset'
+    kwargs:
+        data_split_file_path: '../pretrain.csv'
+        preprocess:
+            - name: 'Clip'
+              kwargs:
+                  mins: -1000
+                  maxs: 1000
```

Comment:
Because there are two modals mr and ct, maybe need two `Clip` with different `mins` and `maxs`.